persistence/test/gov_test.go

```diff
@@ -28,11 +29,181 @@ func TestGetSetParam(t *testing.T) {
 
 	newMaxChains := 42
 
-	err = db.SetParam(types.AppMaxChainsParamName, newMaxChains)
+	err = persistence.SetParam(db, types.AppMaxChainsParamName, newMaxChains)
 	require.NoError(t, err)
 
-	maxChains, err := db.GetMaxAppChains()
+	height, err := db.GetHeight()
+	require.NoError(t, err)
+
+	//maxChains, err := db.GetMaxAppChains(height)
+	maxChains, err := db.GetIntParam(types.AppMaxChainsParamName, height)
+	require.NoError(t, err)
+
+	require.Equal(t, newMaxChains, maxChains)
+}
+
+func TestGetSetStringParam(t *testing.T) {
+	db := persistence.PostgresContext{
+		Height:     0,
+		PostgresDB: testPostgresDB,
+	}
+
+	err := db.InitParams()
+	require.NoError(t, err)
+
+	newServiceNodeMinimumStake := "99999999"
+
+	err = persistence.SetParam(db, types.ServiceNodeMinimumStakeParamName, newServiceNodeMinimumStake)
+	require.NoError(t, err)
+
+	height, err := db.GetHeight()
+	require.NoError(t, err)
+
+	//serviceNodeMinimumStake, err := db.GetParamServiceNodeMinimumStake(height)
+	serviceNodeMinimumStake, err := db.GetStringParam(types.ServiceNodeMinimumStakeParamName, height)
+	require.NoError(t, err)
+
+	require.Equal(t, newServiceNodeMinimumStake, serviceNodeMinimumStake)
+}
+
+func TestGetSetByteArrayParam(t *testing.T) {
+	db := persistence.PostgresContext{
+		Height:     0,
+		PostgresDB: testPostgresDB,
+	}
+
+	err := db.InitParams()
+	require.NoError(t, err)
+
+	newOwner, _ := hex.DecodeString("da034209758b78eaea06dd99c07909ab54c99b44")
+
+	err = persistence.SetParam(db, types.ServiceNodeUnstakingBlocksOwner, newOwner)
+	require.NoError(t, err)
+
+	height, err := db.GetHeight()
+	require.NoError(t, err)
+
+	//owner, err := db.GetServiceNodeUnstakingBlocksOwner(height)
```

Comment:
rm

Reply:
Feel free to resolve the comment if there's no followup or expectation for a response.